spec/model/task_spec.rb

```diff
@@ -16,17 +16,19 @@
     end
     context 'タスクのタイトルと詳細に内容が記載されている場合' do
       it 'バリデーションが通る' do
-        task = Task.new(
-          task_name: 'テストタスク',
-          details: 'テスト詳細')
+        # user = FactoryBot.create(:user)
+        task = FactoryBot.create(:task)
```

Comment:
`FactoryBot.`という記載の部分は設定ファイルに追記することで省略が可能なので、余力があれば調べてみてください。
```suggestion
        task = create(:task)
```